src/main/java/org/apache/skywalking/banyandb/v1/client/Options.java

```diff
@@ -63,6 +63,14 @@ public class Options {
      * SSL: Cert Key Path, BanyanDB server not support mTLS yet
      */
     private String sslKeyPath = "";
```

Comment:
@hanahmily This code seems existing before this PR?

Reply:
Indeed. The initial commit of this file introduced it. Thanks to @wankai123, he indicated that "BanyanDB server does not support mTLS yet," which led me to understand that the client does not require them.